model/project_parser.go

```diff
@@ -534,9 +538,12 @@ func LoadProjectForVersion(v *Version, id string, shouldSave bool) (ProjectInfo,
 	if err != nil {
 		return ProjectInfo{}, errors.Wrap(err, "error loading project")
 	}
-	pc, err := CreateProjectConfig([]byte(v.Config), id)
-	if err != nil {
-		return ProjectInfo{}, errors.Wrap(err, "error loading project config")
+	var pc *ProjectConfig
```

Comment:
Sorry I just realized: this is actually the legacy case  (i.e. before parser project got created). So similar to how we mention we don't handle opts for this (mentioned at line 535), we don't really need to handle project config for this. We do maybe need to handle it at 518? But we could just look it up from the DB, not read it from v.Config.  

(Although I'm not sure if LoadProjectInto needs to populate config)